controllers/admin/AdminMetaController.php

```diff
@@ -746,7 +746,9 @@ public function getRobotsContent()
         );
 
         // Directories
-        $tab['Directories'] = array('classes/', 'config/', 'download/', 'mails/', 'modules/', 'translations/', 'tools/');
+        $tab['Directories'] = array('Adapter/', 'admin/', 'cache/', 'cgi-bin/', 'classes/', 'config/', 'controllers/',
+            'Core/', 'css/', 'download/', 'img/', 'js/', 'localization/', 'log/', 'mails/', 'override/',
+            'pdf/', 'themes/', 'tools/', 'translations/', 'upload/', 'webservice/');
```

Comment:
Hello @Quetzacoalt91 

Why did you put `'Adapter'`, `'Core'` and not `'src'` directly (contain adapter, core, and prestashopbundle) ?

Why didn't you put `'app'`, `'bin'`, `'modules'`, `'test'`, `'web`' ?

Why did you put `'img'` and `'upload'` (I think medias (cms) are uploaded into uploaded ?) ?
I don't know if images must be disalow ?